axelor-bank-payment/src/main/java/com/axelor/apps/bankpayment/service/bankreconciliation/BankReconciliationService.java

```diff
@@ -1087,4 +1088,50 @@ public void reconcileSelected(BankReconciliation bankReconciliation) throws Axel
         bankReconciliationLineService.reconcileBRLAndMoveLine(
             bankReconciliationLine, moveLines.get(0));
   }
+
+  public String getDomainForWizard(
+      BankReconciliation bankReconciliation,
+      BigDecimal bankStatementCredit,
+      BigDecimal bankStatementDebit) {
+    if (bankReconciliation != null
+        && bankReconciliation.getCompany() != null
+        && bankStatementCredit != null
+        && bankStatementDebit != null) {
+      String query =
+          "(self.bankReconciledAmount < self.debit or self.bankReconciledAmount < self.credit)"
+              + " AND ((self.debit > 0 and "
+              + bankStatementCredit.compareTo(BigDecimal.ZERO)
```

Comment:
I know it was like that in action-bank-reconciliation-attrs-move-line-domain-on-wizard but it useless right? 
Maybe just add bankStatementCredit.signum() > 0 and bankStatementDebit.signum() > 0 in the previous if?